packages/optimistic-oracle/index.js

```diff
@@ -17,6 +17,17 @@ const { OptimisticOracleProposer } = require("./src/proposer");
 const { getWeb3 } = require("@uma/common");
 const { getAbi, getAddress } = require("@uma/contracts-node");
 
+// Types of oracles, used by the optimistic oracle proposer off-chain bot to determine if a price request has resolved.
+// Note: Keys and valuesmust match the contract names listed in core/networks file so that this file can use them
+// as input into getAbi and getAddress.
+const OracleType = {
+  Voting: "Voting", // Used on mainnet when optimistic oracle directly submits price requests to Voting.
+  OracleChildTunnel: "OracleChildTunnel", // Used in production when running proposer bot on sidechain that needs to
+  // bridge price requests back to L1.
+  MockOracleAncillary: "MockOracleAncillary", // Used for testing when caller wants to be able to manually push prices
```

Comment:
Feels a little odd to tell the bot/test that it's interacting with a test contract, but if it's just used for logging/debugging, I think it's probably fine.